theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/EasingOption.tsx

```diff
@@ -59,8 +58,8 @@ type IProps = {
 
 const EasingOption: React.FC<IProps> = React.forwardRef((props, ref) => {
   const [tooltip, tooltipHostRef] = useTooltip(
-    {enabled: true, verticalPlacement: props.tooltipPlacement},
-    () => <EasingTooltip>{props.easing.label}</EasingTooltip>,
+    {enabled: true, verticalPlacement: props.tooltipPlacement, verticalGap: 0},
+    () => <EasingTooltip showArrow={false}>{props.easing.label}</EasingTooltip>,
```

Comment:
Suggestion: Maybe rename `showArrow` to `showPopoverEdgeTriangle={false}`. Generally, just confused by whether there were arrows within the easing presets preview SVG.

Additionally, apply this rename to `popover`.

Reply:
renamed in `BasicPopover`, `usePopover` does not use this name so I am not sure what you mean by your second paragraphg.